src/main/scala/no/gdl/bookapi/BookSwagger.scala

```diff
@@ -27,5 +27,5 @@ object BookApiInfo {
 }
 
 class BookSwagger extends Swagger("2.0", "0.8", BookApiInfo.apiInfo) {
-  addAuthorization(OAuth(List("books:all"), List(ApplicationGrant(TokenEndpoint("/auth/tokens", "access_token")))))
-}
+//  addAuthorization(OAuth(List("books:all"), List(ImplicitGrant(LoginEndpoint(BookApiProperties.LoginEndpoint), "access_token"))))
```

Comment:
Tanken med å ha denne her som en kommentar er for fremtidig bruk?
Kan godt kommentere hvorfor denne er her, men kommentert ut.

Reply:
Det var tanken. Om vi skulle få behov for å innføre skriving via apiet seinare. Men eg kan godt ta det vekk, også fra BookApiProperties.